scripts/new_project.kts

```diff
@@ -16,6 +17,14 @@ object NewProject {
     private const val PATTERN_APP = "^([A-Z][a-zA-Z0-9\\s]*)|([a-z][a-z0-9-]*)$"
     private const val PATTERN_PACKAGE = "^[a-z]+(\\.[a-z][a-z0-9]*)+$"
 
+    private val helpMessage = """
+Run kscript new_project.kts to create a new project with the following arguments:
+    package-name=   New package name (i.e., com.example.package)
+    app-name=       New app name (i.e., MyApp, "My App", "my-app")
+
+Example: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
+    """.trimIndent()
```

Comment:
I think @Wadeewee's comment about readability here meant to indent the actual message so it isn't aligned with the left most column, that's why we added the `.trimIndent()` call to it 😅 
```suggestion
    private val helpMessage = """
        Run kscript new_project.kts to create a new project with the following arguments:
            package-name=   New package name (i.e., com.example.package)
            app-name=       New app name (i.e., MyApp, "My App", "my-app")
        
        Example: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
    """.trimIndent()
```